apps/science/include/CentrifugeConfig.h

```diff
@@ -14,19 +14,21 @@
 namespace Centrifuge {
 
 namespace Internal {
-static Actuator::DCMotor motor(MTR_PWM_1, MTR_DIR_1, false);
+static Actuator::DCMotor motor(MTR_PWM_CENTRIFUGE, MTR_DIR_CENTRIFUGE, false);
 
-// Felix TODO: ensure that Centrifuge uses AEAT6012 encoder. If not please correct
-static Encoder::AEAT6012 encoder({NC, NC, NC, 0});  // Felix TODO: Fill in correct pins for Centrifuge
+// clk, miso, cs
+static Encoder::AEAT6012 encoder({CENTRIFUGE_ENC_SCK, CENTRIFUGE_ENC_MISO, CENTRIFUGE_ENC_CS, 0});
 
 static PID::PID velPID({1, 0, 0, -1, 1, 0, false, false});
 static PID::PID posPID({1, 0, 0, -1, 1, 0, false, false});
 
-constexpr float maxDegPerSec = std::numeric_limits<float>::infinity();  // TODO: figure out maxDegPerSec of motors (601?)
+constexpr float maxDegPerSec = 601;                                     // 10.5 RAD/s
 constexpr float maxCurrent   = std::numeric_limits<float>::infinity();  // since no current sensor
 
-static Controller::Position pos(&motor, &encoder, std::nullopt, &posPID, maxDegPerSec, maxCurrent, LIM_SW_1, LIM_SW_2);
-static Controller::OpenLoop open(&motor, &encoder, std::nullopt, maxDegPerSec, maxCurrent, LIM_SW_1, LIM_SW_2);
+static Controller::Position pos(&motor, &encoder, std::nullopt, &posPID, maxDegPerSec, maxCurrent, CENTRIFUGE_SW_1,
+                                CENTRIFUGE_SW_2);
```

Comment:
nitpick but please have the LIM part in these. without LIM, SW sounds like software